.vscode/tasks.json

```diff
@@ -7,22 +7,23 @@
 // ${cwd}: the current working directory of the spawned process
 // A task runner that calls a custom npm script that compiles the extension.
 {
-	"version": "0.1.0",
-	// we want to run npm
-	"command": "npm",
-	// the command is a shell script
-	"isShellCommand": true,
-	// show the output window only if unrecognized errors occur.
-	"showOutput": "silent",
-	// we run the custom script "compile" as defined in package.json
-	"args": [
-		"run",
-		"compile",
-		"--loglevel",
-		"silent"
-	],
-	// The tsc compiler is started in watching mode
-	"isBackground": true,
-	// use the standard tsc in watch mode problem matcher to find compile problems in the output.
-	"problemMatcher": "$tsc-watch"
-}
\ No newline at end of file
+	"version": "2.0.0",
+	"task": [
+		{
+			"label": "build",
+			"command": "npm",
+			"type": "shell",
+			"presentation": {
+				"reveal": "never"
+			},
+			"args": [
```

Comment:
Since you're updating the task, can you add:
```
"group": {
	"kind": "build",
	"isDefault": true
},
```

If you open up the command palette:
![screen shot 2018-10-19 at 10 15 24 am](https://user-images.githubusercontent.com/11282622/47233519-e886e380-d387-11e8-8bdf-f3ef137cf408.png)

This designates this task as the build task to be run as a part of this:
![screen shot 2018-10-19 at 10 14 43 am](https://user-images.githubusercontent.com/11282622/47233506-d86f0400-d387-11e8-9094-2015d52bf66d.png)